guides/common/modules/proc_enrolling-server-with-the-ad-server.adoc

```diff
@@ -18,7 +18,8 @@ Installing GSS-proxy and {nfs-client-package}:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# {project-package-install} sssd adcli realmd ipa-python-compat krb5-workstation samba-common-tools
+# {project-package-install} adcli ipa-python-compat krb5-workstation \
```

Comment:
When I run this on EL9 (which we're supposed to support now):
```console
# dnf install -q adcli ipa-python-compat krb5-workstation oddjob oddjob-mkhomedir realmd samba-common-tools sssd
Error: Unable to find a match: ipa-python-compat
```

Reply:
Oh well. We shouldn't even be listing these packages on the Foreman side because required packages do change over time. Ideally, I'd like to link to the RHEL docs for the up-to-date list but I can't because the procedure on the Foreman side is a blend of of the [SSSD-based joining](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/integrating_rhel_systems_directly_with_windows_active_directory/index#discovering-and-joining-an-ad-domain-using-sssd_connecting-directly-to-ad) and the [Samba-based joining](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/integrating_rhel_systems_directly_with_windows_active_directory/index#proc_joining-samba-to-a-domain_connecting-rhel-systems-directly-to-ad-using-samba-winbind) so I don't know which of the two scenarios I should link to.

Even more ideally, I'd like to link to the RHEL docs for the whole procedure of joining an AD domain. I can't for the same reason.

Reply:
In any case, I checked the RHEL docs for versions 7, 8, and 9 and neither lists ipa-python-compat as required for AD integration. And I believe it was replaced in RHEL 8. Based on that, I believe we can just drop it.